### PR TITLE
Refactor/business logic model names

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: mobile/pubspec.lock [hunk 1]
```diff
@@ -165,10 +165,10 @@ packages:
     dependency: transitive
     description:
       name: collection
-      sha256: "4a07be6cb69c84d677a6c3096fcf960cc3285a8330b4603e0d463d15d9bd934c"
+      sha256: cfc915e6923fe5ce6e153b0723c753045de46de1b4d63771530504004a45fae0
       url: "https://pub.dev"
     source: hosted
-    version: "1.17.1"
+    version: "1.17.0"
```
Comment (bonomat) [inline mobile/pubspec.lock:171]: I don't think this change belongs in here

Comment (da-kami) [reply mobile/pubspec.lock:171]: :/ this keeps happening, I think it's related to #230

**Review by bonomat — APPROVED** [review]
Nice :)

**Pull request merged**